eq-author-api/package.json

```diff
@@ -9,7 +9,8 @@
     "lint": "eslint .",
     "test": "./scripts/test.sh",
     "test:coverage": "yarn test --coverage",
-    "test:breakingChanges": "node scripts/checkForBreakingChanges.js"
+    "test:breakingChanges": "node scripts/checkForBreakingChanges.js",
+    "knex": "knex --knexfile knexfile.js --cwd ."
```

Comment:
Seem to recall the `knex` command being removed because the app runs migrations on initialisation. Is the re-instatement of this command to help with development?

Reply:
To run migrations as per migrations [README](https://github.com/ONSdigital/eq-author-app/tree/master/eq-author-api)